mmv1/third_party/terraform/tests/resource_storage_bucket_test.go.erb

```diff
@@ -766,6 +767,30 @@ func TestAccStorageBucket_encryption(t *testing.T) {
 	})
 }
 
+<% unless version == "ga" -%>
+func TestAccStorageBucket_publicAccessPrevention(t *testing.T) {
+	t.Parallel()
+
+	bucketName := fmt.Sprintf("tf-test-acl-bucket-%d", randInt(t))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccStorageBucket_publicAccessPrevention(bucketName, "enforced"),
+			},
+			{
+				ResourceName:            "google_storage_bucket.bucket",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"force_destroy"},
+			},
+		},
+	})
+}
+<% end -%>
+
```

Comment:
```suggestion
func TestAccStorageBucket_publicAccessPrevention(t *testing.T) {
	t.Parallel()

	bucketName := fmt.Sprintf("tf-test-acl-bucket-%d", randInt(t))

	vcrTest(t, resource.TestCase{
		PreCheck:  func() { testAccPreCheck(t) },
		Providers: testAccProviders,
		Steps: []resource.TestStep{
			{
				Config: testAccStorageBucket_publicAccessPrevention(bucketName, "enforced"),
			},
			{
				ResourceName:            "google_storage_bucket.bucket",
				ImportState:             true,
				ImportStateVerify:       true,
				ImportStateVerifyIgnore: []string{"force_destroy"},
			},
		},
	})
}

```

Reply:
See remarks below. Why would we want to run this test for the GA provider, when the functionality is only being added to the beta one?

Reply:
Ah. Is this because the tests are specifying the provider per-resource, so the beta provider will be used for this test? If so, I think I get it... will remove that gating.